docs/nodes/qgb-orchestrator.md

```diff
@@ -0,0 +1,303 @@
+# QGB Orchestrator
+
+<!-- markdownlint-disable MD013 -->
+<!-- markdownlint-disable MD010 -->
+
+The role of the orchestrator is to sign attestations using its corresponding validator EVM private key. These attestations are generated within the QGB module of the Celestia-app state machine. To learn more about what attestations are, you can refer to this [link](https://github.com/celestiaorg/celestia-app/tree/main/x/qgb).
+
+## How it works
+
+The orchestrator does the following:
+
+1. Connect to a Celestia-app full node or validator node via RPC and gRPC and wait for new attestations
+2. Once an attestation is created inside the QGB state machine, the orchestrator queries it.
+3. After getting the attestation, the orchestrator signs it using the provided EVM private key. The private key should correspond to the EVM address provided when creating the validator. More details in [here](https://docs.celestia.org/nodes/validator-node/#setup-qgb-keys).
+4. Then, the orchestrator pushes its signature to the P2P network it is connected to, via adding it as a DHT value.
+5. Listen for new attestations and go back to step 2.
+
+The orchestrator connects to a separate P2P network than the consensus or the data availability one. So, we will provide bootstrappers for that one.
+
+This means that even if the consensus node is already connected to the consensus network, if the orchestrator doesn't start with a list of bootstrapper to its specific network, then, it will not work and will output the following logs:
+
+```text
+I[2023-04-26|00:04:08.175] waiting for routing table to populate        targetnumberofpeers=1 currentcount=0
+I[2023-04-26|00:04:18.175] waiting for routing table to populate        targetnumberofpeers=1 currentcount=0
+I[2023-04-26|00:04:28.175] waiting for routing table to populate        targetnumberofpeers=1 currentcount=0
+```
+
+## How to run
+
+### Install the QGB binary
+
+Make sure to have the QGB binary installed. Check [here](https://github.com/celestiaorg/orchestrator-relayer/blob/main/README.md) for more details.
```

Comment:
same here, re: show folks in the future how to install the binary within the docs.